### PR TITLE
Add sysinfo opt out prompt

### PR DESCRIPTION
**Pull request by pchote**
This displays an opt-out dialog the first time the game is run (after each change to the data collection):

### CODE REVIEW

**Comment by obrakmann** [comment]
Looks ok to me, :+1: 

Will need the server-side stuff, though, before we release it into the wild.

**Comment by abcdefg30** [comment]
The scrollbar still looks a bit weird to me, why do we have it if we can't scroll and don't use line breaks either (one line even extends the horizontal width)? Just minor visual polish though.

**Comment by Phrohdoh** [comment]
On the topic of nits:
`s/.NET Runtime/Runtime/` is a bit more true to Mono and may cut down on the length of that line enough to prevent clipping.

**Comment by pchote** [comment]
> Will need the server-side stuff, though, before we release it into the wild.

This is compatible with the current server-side stuff.  The changes I talked about in IRC are in addition to what it does now.

**Comment by pchote** [comment]
Mono *is* a .NET runtime, so I don't think it should be changed.

File: OpenRA.Mods.Common/Widgets/Logic/MainMenuLogic.cs [hunk 1]
```diff
@@ -32,6 +36,22 @@ protected enum MenuType { Main, Singleplayer, Extras, MapEditor, None }
...
+		const int SystemInformationVersion = 1;
+		Dictionary<string, Pair<string, string>> GetSystemInformation()
+		{
+			var lang = System.Globalization.CultureInfo.InstalledUICulture.TwoLetterISOLanguageName;
+			return new Dictionary<string, Pair<string, string>>()
+			{
+				{ "id", Pair.New("Anonymous ID", Game.Settings.Debug.UUID) },
+				{ "platform", Pair.New("OS Type", Platform.CurrentPlatform.ToString()) },
+				{ "os", Pair.New("OS Version", Environment.OSVersion.ToString()) },
+				{ "runtime", Pair.New(".NET Runtime", Platform.RuntimeVersion) },
```
Comment (Mailaender) [inline OpenRA.Mods.Common/Widgets/Logic/MainMenuLogic.cs]: `.NET Runtime: .NET CLR 4.0.30319.42000` will be redundant though.

Comment (pchote) [inline OpenRA.Mods.Common/Widgets/Logic/MainMenuLogic.cs]: The .NET Runtime used on windows is the .NET CLR 4.0.x.  There's no redundancy here.

**Comment by obrakmann** [comment]
I already added the changelog entry to [the Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog)

**Comment by reaperrr** [comment]
Looks good to me :+1:

**Pull request merged**